fluss-server/src/test/java/com/alibaba/fluss/server/coordinator/TableManagerTest.java

```diff
@@ -159,6 +172,9 @@ void testDeleteTable() throws Exception {
         assertThat(zookeeperClient.getTableAssignment(tableId)).isEmpty();
         // the table will also be removed from coordinator context
         assertThat(coordinatorContext.getAllReplicasForTable(tableId)).isEmpty();
+
+        // the table's remote kv root dir should be removed.
+        assertThat(remoteStorageHandler.isTableKvDirExists(DATA1_TABLE_PATH_PK, tableId)).isFalse();
```

Comment:
Also add IT to make sure drop table will also delete kv remote dir.
Maybe we can add in `KvSnapshotITCase`